wordpress-mobile/WordPress-Android
# Clean up after legacy editor

This PR removes some left-overs from the removal of legacy and hybrid editors.

It might be easier to review one commit at a time.

To test:
1. Open the editor
2. Open the post settings
3. Edit the excerpt
4. Save the changes and make sure the excerpt got uploaded to the server

PR submission checklist:
